source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,570 @@
+
+`Supported Devices/Mobile Device Requirements <https://docs.mattermost.com/install/requirements.html#mobile-apps>`_
+    Basic mobile device requirements are provided in the link above.
+
+********************************
+Public App Store vs. Custom Apps
+********************************
+
+The most critical decision you will make in your mobile deployment will be whether to use the apps provided by Mattermost via Google Play and Apple App Store, or to build and distribute custom versions of the Mattermost apps. Below we provide a very general overview of these options.
+
+Public App Store
+================
+
+Utilizing the app distributed by Mattermost in Google play and the Apple App Store greatly reduces the deployment time and is our recommended approach. Key benefits include:
+
+- Easier deployment, driven by user needs.
+- Ability to use the Mattermost Push Notification Service, or MPNS.
+- Apps automatically updated with the latest features, enhancements and security updates.
+
+Again, this is the recommended route which is as simple as downloading the Mattermost mobile application from the corresponding store, entering a URL to point towards your Mattermost server, and logging in.
+
+Custom Apps
+===========
+
+If you desire to customize the applications, or do not want your users downloading the application from the public app stores, you will need to build the apps yourself. 
+
+As the apps are an open source project, it will require a fork, and your team will be responsible for maintaining the fork, as well as keeping the fork up to date with any changes made by Mattermost.
+
+This process can be complicated. It will also greatly increase deployment time, not only initially, but whenever the apps need to be updated. 
+
+We recommend having your development team take a look at the documentation to ensure they understand the scale and requirements of this path. In general, this route will present some challenges, including:
+
+- Obtaining/providing certificates for your custom application
+- Signing your custom applications
+- Distributing your applications via public or private app stores
+
+*******************************************
+Mattermost Push Notification Service (MPNS)
+*******************************************
+
+Receiving notifications on a mobile device is a core value of your mobile deployment.
+
+It also represents a return on your deployment investment through better-connected users. 
+
+A push proxy is a key technology behind notification transmission. It enables notifications between the server and the mobile application.
+
+Mattermost provides a self-hosted Push Proxy you can deploy, the Mattermost Push Notification Service (MPNS). This is also available via a hosted-by-Mattermost option, or Hosted Push Notification Service (HPNS)*.
```

Comment:
I'm not sure what the * relates to, as there is no corresponding 'footnote' with an * as an explanation (as per the EMM providers one below).  If it is the _note_ below then I assume it is not needed?

Reply:
+1